modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -1,17 +1,14 @@
-= Refining Batch Steps Processing
+= Using Batch Processing Scopes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-aliases: batch-aggregator-concept.adoc, filter-records-batch-faq.adoc
 
-You can refine the work that a batch step performs upon the records it processes.
+You can set up and configure Mule batch processing scopes to perform common batch processing use cases and to enhance the performance of batch jobs for cases in which the defaults require adjustment.
```

Comment:
```suggestion
Mule batch processing components designed to manage bulk processing of records with minimal adjustments to their default settings. However, the scopes are highly configurable so that batch jobs meet specific processing and performance requirements. You can set up and configure Mule batch processing scopes to perform common batch processing use cases and to enhance the performance of batch jobs for cases in which the defaults require adjustment.
```

Reply:
@IvanAndresFritzler here's a revision of the intro paragraph